postgres_db/src/download_queue.rs

```diff
@@ -272,20 +273,42 @@ pub fn load_chunk_next(
 }
 
 pub fn update_from_tarballs(conn: &DbConnection, tarballs: &mut Vec<DownloadedTarball>) {
```

Comment:
Actually, now that the downloading stuff is decoupled from the db, i think we might want to remove the mutation logic here, and handle it in the downloader. It makes more sense to me. Also the `if !tarballs.is_empty()` expr

Reply:
\* Referring to the update_from_tarballs function, i tagged the wrong thing

Reply:
sure go for it with another PR